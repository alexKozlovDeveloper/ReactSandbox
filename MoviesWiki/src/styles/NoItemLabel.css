.title {
    color: white;
    font-family: cursive;
    text-align: center;
    height: 500px;
    font-size: 30px;
}