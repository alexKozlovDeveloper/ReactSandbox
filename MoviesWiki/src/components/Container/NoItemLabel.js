import React, { Component } from "react";

import styles from "../../styles/NoItemLabel.css"

const NoItemLabel = () => {
    return  <div className={styles.title}>
                No films found
            </div>;
}

export default NoItemLabel;