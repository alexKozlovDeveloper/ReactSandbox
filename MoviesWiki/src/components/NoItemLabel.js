import React, { Component } from "react";

import styles from "../styles/NoItemLabel.css"

function NoItemLabel(props) {
    return  <div className={styles.title}>
                No films found
            </div>;
}

export default NoItemLabel;