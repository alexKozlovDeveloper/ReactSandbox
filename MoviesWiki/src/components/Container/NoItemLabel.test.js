import React from 'react';
import { shallow, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import NoItemLabel from "./NoItemLabel";

configure({adapter: new Adapter()});

test('NoItemLabel rendering test', () => {

    const component = shallow(<NoItemLabel />);
    expect(shallowToJson(component)).toMatchSnapshot();
 });

