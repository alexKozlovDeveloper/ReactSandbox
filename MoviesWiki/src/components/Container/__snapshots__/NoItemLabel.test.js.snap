// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoItemLabel rendering test 1`] = `
<div
  className="title"
>
  No films found
</div>
`;
